<?xml version="1.0"?>
<launch>
  <!-- start novate_oemv gps node -->
  <node pkg="novatel_oemv" type="gps_node" name="gps_node" output="screen">
    <!-- param name="port" value="/dev/ttyUSB_GPS" / -->
    <param name="port" value="/dev/ttyUSB0" />
    <!-- NORMAL, EGNOS, PDPFILTER or GL1DE -->
    <param name="mode" value="GL1DE"/> 
  </node>
  <!-- node pkg="novatel_oemv" type="gps_node" name="gps_node" output="screen"/ -->    

  <!-- gps_conv - utm to xyz for ekf -->
  <include file="$(find novatel_oemv)/launch/gps_conv.launch"/>

</launch>

